litmus-portal/graphql-server/graph/schema.resolvers.go

```diff
@@ -43,11 +44,40 @@ func (r *mutationResolver) ReRunChaosWorkFlow(ctx context.Context, workflowID st
 }
 
 func (r *mutationResolver) CreateUser(ctx context.Context, user model.CreateUserInput) (*model.User, error) {
+	return usermanagement.CreateUser(ctx, user)
+}
+
+func (r *mutationResolver) CreateProject(ctx context.Context, projectName string) (*model.Project, error) {
+	//fetching all the user's details from jwt token
 	claims := ctx.Value(authorization.UserClaim).(jwt.MapClaims)
 	userUID := claims["uid"].(string)
 	role := claims["role"].(string)
+	email := claims["email"]
+	fullname := claims["name"]
+	username := claims["username"].(string)
```

Comment:
No need to take all the claims from the token now, it would be better to take the uid and query the user collection and get the rest of the details just to make sure the user actually exists in the gql db